packages/dashboard/src/components/pages/platforms/dmt5_synthetic/dmt5-synthetic.scss

```diff
@@ -0,0 +1,28 @@
+.dw-about-us {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    margin: 0 auto;
+    flex-direction: column;
+    background: var(--general-section-1);
+}
+
+.dw-about-us__text {
+    width: 80%;
+}
+
+.dw-about-us__title {
+    margin-top: 3.2rem;
+    margin-bottom: 2.4rem;
+}
+
+.dw-about-us__subtitle {
+    margin-bottom: 4rem;
+}
+
+.dw-about-us__wrapper {
+    display: grid;
+    grid-template-columns: repeat(auto-fit, minmax(200px, 1fr));
+    grid-gap: 1.6rem;
+    width: 80%;
+}
```

Comment:
Use the power of Sass nesting when you have it. 🚀

Reply:
this was a remnant from previous about us, will remove the entire file 😅